src/sentry/runner/commands/cleanup.py

```diff
@@ -767,7 +760,7 @@ def run_bulk_deletes_by_project(
                         order_by=order_by,
                     )
 
-                    for chunk in q.iterator(chunk_size=100):
+                    for chunk in q.iterator(chunk_size=100, batch_size=30000):
```

Comment:
The default is 10k:
https://github.com/getsentry/sentry/blob/43e429712e0772a3a606807a1ed30e2f754903d9/src/sentry/db/deletion.py#L94

Let's try doing 3x and see if things improve.